src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setSearch] = useState('');
```

Comment:
Let's follow the naming convention for the useState. 
`const [value, setValue] = useState()`